new/LOC7400Waramit.xml

```diff
@@ -0,0 +1,64 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LOC7400Waramit" xml:lang="en" type="place">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Warāmit</title>
```

Comment:
same as LOC6979Waramit see https://github.com/BetaMasaheft/Places/pull/100?

Reply:
Yes, how can I discard LOC7400Waramit? We don't need this new ID any more.

Reply:
You will have to delete it in a separate PR after this one is merged

Reply:
OK.

Reply:
Sorry for a late intrusion into the discussion, I would nearly certainly read this word as ይዝማ; even in view of the absence of another ይ፣ yet multiple examples of ደ, written differently.  ይዝማ exists also today as locality, you can find it in the internet, cp. `https://en.wikipedia.org/wiki/Dibo`

Reply:
> I would nearly certainly read this word as ይዝማ; even in view of the absence of another ይ፣ yet multiple examples of ደ, written differently. ይዝማ exists also today as locality, you can find it in the internet, cp. https://en.wikipedia.org/wiki/Dibo

Yes, very convincing - even though it is more than 100km away from Kəbrān!